### PR TITLE
Refactor; implement grid display

### LINKED ISSUE #14: US-2a: As a player, I want to be able to see the current placement of the buildings so that I can plan my next step

### PR DESCRIPTION
**Pull request by CaeNGnp**
* Makes some refactorings to make way for multiple buildings.
* Adds display to start visual testing for QA

Resolves #14

### CODE REVIEW

File: SimpCity/Game.cs [hunk 1]
```diff
@@ -5,16 +5,26 @@
 namespace SimpCity {
     public enum BuildingTypes {
         Beach,
+        Factory,
+        Highway,
+        House,
+        Shop
```
Comment (CaeNGnp) [inline SimpCity/Game.cs]: Remove these

**Review by jordie007 — COMMENTED** [review]
The code looks alright to me.

**Comment by JSQJ** [comment]
Missing lines in display in game. slightly different from case study.

**Review by JSQJ — APPROVED** [review]
No issues with the code. Only missing some lines in the display while in game

**Comment by CaeNGnp** [comment]
Ok this should be ready for merge now @jordie007, the fix for missing lines will come in in another PR.

**Pull request merged**